src/components/House.js

```diff
@@ -189,9 +189,10 @@ class House extends React.Component {
     Howler.volume(0);
     window.clearInterval(this.gameIntervalID);
     window.setTimeout(() => {
-       alert("GAME OVER. Your house has become awful");
-       window.setTimeout(() => {this.props.setGameState('menu');}, 2000);
+       document.getElementById('House-Id').innerHTML = "GAME OVER.&#13;&#10; Your house has become awful.";
```

Comment:
We should change this text with react instead of setting innerhtml.

Reply:
Are you trying to use line breaks with the `&#13;` stuff? You have to use `<br>`